httpbin/handlers.go

```diff
@@ -981,3 +1006,21 @@ func (h *HTTPBin) Bearer(w http.ResponseWriter, r *http.Request) {
 	})
 	writeJSON(w, body, http.StatusOK)
 }
+
+// Returns a new rand.Rand from the given seed string.
+func parseSeed(rawSeed string) (*rand.Rand, error) {
```

Comment:
Thanks for factoring out this functionality!  Would you mind moving this into `helpers.go` alongside the other small utility functions?